src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -1152,6 +1151,15 @@ int fx_muxer_t::read_config_and_execute(
     }
 
     auto app_config = app->get_runtime_config();
+    bool is_framework_dependent = app_config.get_is_framework_dependent();
+
+    // These settings are only valid for framework-dependent apps
+    if (is_framework_dependent)
+    {
+        fx_version_specified = get_last_known_arg(opts, opts_fx_version, _X(""));
```

Comment:
Similar to above - I don't think it is possible for apphost to take ANY command line arguments.  The host arguments can only ever work with the `dotnet.exe` muxer.